lib/flipper/api/error_response.rb

```diff
@@ -24,6 +24,7 @@ def as_json
         feature_not_found: Error.new(1, "Feature not found.", "", 404),
         group_not_registered: Error.new(2, "Group not registered.", "", 404),
         percentage_invalid: Error.new(3, "Percentage must be a positive number less than or equal to 100.", "", 400),
+        flipper_id_invalid: Error.new(4, "Required parameter flipper_id cannot be nil.", "", 400),
```

Comment:
What do you think about "Required parameter flipper_id is missing."? I'm not sure that we need to mention nil.